src/com/github/johnthagen/cppcheck/CppcheckInspection.java

```diff
@@ -214,19 +214,23 @@ private static String executeCommandOnFile(@NotNull final String command,
         }
 
         if (output.isTimeout()) {
-            throw new ExecutionException("Cppcheck Error: Timeout: " + cmd.getCommandLineString());
+            throw new CppcheckError("Timeout\n"
+                    + cmd.getCommandLineString());
         }
 
         if (output.getExitCode() != 0) {
-            throw new ExecutionException("Cppcheck Error: Exit Code - " + output.getExitCode() + " : " +
+            throw new CppcheckError("Exit Code " + output.getExitCode() + "\n" +
+                    "StdOut: " + output.getStdout() + "\n" +
                     cmd.getCommandLineString());
         }
 
         if (cppcheckMisraPath != null && !cppcheckMisraPath.isEmpty()) {
             if (output.getStdout().contains("Bailing out from checking")) {
                 // MISRA Mode and something went wrong with the misra addon
-                throw new ExecutionException("Cppcheck MISRA Bail: " + cmd.getCommandLineString() +
-                        "\n StdOut: \n" + output.getStdout() + "\n StdErr: " + output.getStderr());
+                throw new CppcheckError("MISRA Bail\n" +
+                        cmd.getCommandLineString() + "\n" +
+                        "StdOut: " + output.getStdout() + "\n"+
+                        "StdErr: " + output.getStderr());
```

Comment:
Let's use the canonical `stderr` casing.

Reply:
Will do - I chose the casing from the exiting messages.

Reply:
done